### ISSUE TITLE
The rspec matcher does not make narrative intent clear

### ISSUE THREAD

**Issue by tastycode** [somebody32/adequack#1]
When you see "be_adequack_to" in test, the meaning is not obvious from a narrative point of view. If tests are documentation, then the reader of the test will not understand the intent without exploring the gem. The gem should find a better name: 

A couple of suggestions? 
- DogClass.should expose_the_interface_of(DogInterface) 
- DogClass.should act_like(DogInterface) 
- DogClass.should be_only_mockable_against(DogInterface)

**Comment by somebody32** [comment]
Hi, 

good point, thanks for the suggestion!
I'll try to find a better helper name and deprecate the previous one for the next release.

**Comment by swrobel** [comment]
Great suggestion, and @Somebody32 really love what you're doing here, I've always found this to be an uncomfortable hole in my test suite.

**Comment by somebody32** [comment]
Hi, 
according to this issue I think that I'll code these helpers:

`be_adequack_to` will be replaced with `behave_like`

``` ruby
DogClass.should behave_like DogInterface/DogObject
```

and `adequack_double` -> `behavioral_double`

``` ruby
let(:dog) { behavioral_double double, DogInterface }
```

I'll deprecate the previous ones in the next release. 
Any feedback will be highly appreciated

**Comment by swrobel** [comment]
:+1:

**Comment by somebody32** [comment]
Closed in version 0.0.3. Thanks to everyone!
